robyn/templating.py

```diff
@@ -28,5 +29,10 @@ def render_template(self, template_name, **kwargs) -> Response:
             headers=Headers({"Content-Type": "text/html; charset=utf-8"}),
         )
 
+    def add_template_global(self, func: Callable, name: str | None = None):
```

Comment:
Hey @JarriqTheTechie 👋 

Just two suggestions 
- can you write a docstring for this function?
- can you add docs for this function in `docs_src` folder?